python/create_skill_tree_svg.py

```diff
@@ -29,8 +53,9 @@ def process_svg(template, data):
                 if j >= 5:
                     continue
             value = data["row"][i][j]
+            splitted_string = make_text_multiline(list(value))
```

Comment:
nit: I'd opt for `split_string` here

Reply:
Oopsie, I'll try to make more changes on the script and fix this then.